scripts/package-tools.sh

```diff
@@ -1,177 +1,56 @@
 #!/bin/bash
 set -e -x -o pipefail
 
-BIN_DIR=${BIN_DIR:-./bin}
-
-# Check if TOOL_REGISTRY_REPOS is set and non-empty
-if [[ -z "${TOOL_REGISTRY_REPOS}" ]]; then
-    echo "Error: TOOL_REGISTRY_REPOS environment variable is not set or is empty."
-    exit 1
-fi
-
-if [[ -n "${GITHUB_TOKEN}" ]]; then
-    set +x
-    git config --global url."https://${GITHUB_TOKEN}@github.com/".insteadOf "https://github.com/" 2>/dev/null
-    set -x
-fi
-
-cd $(dirname $0)/..
-
-if [ ! -e obot-tools ]; then
-    mkdir obot-tools
-fi
-cd obot-tools
-
-# Convert TOOL_REGISTRY_REPOS into an array by replacing commas with spaces
-read -r -a TOOL_REPOS <<< "${TOOL_REGISTRY_REPOS//,/ }"
-REGISTRY_REMAP=()
-LOCAL_REGISTRIES=()
-OBOT_SERVER_VERSIONS=""
-
-# Iterate over the repositories
-for REPO in "${TOOL_REPOS[@]}"; do
-    # Extract the repo name (e.g., tools, enterprise-tools)
-    REPO_NAME=$(basename "${REPO}")
-    HASH=""
-    # If there is a hash in the repo name, then extract it
-    if [[ "${REPO_NAME}" == *"@"* ]]; then
-      # Extract the part after "@"
-      HASH="${REPO_NAME#*@}"
-      REPO_NAME=${REPO_NAME%@*}
-      REPO=${REPO%@*}
-	fi
-    REPO_DIR="obot-tools/${REPO_NAME}"
-
-    # Clone the repository into the target directory
-    echo "Cloning ${REPO} into ${REPO_DIR}..."
-    if git clone "https://${REPO}" "${REPO_NAME}"; then
-    	# Checkout the commit, if one was set.
-    	if [[ -n "${HASH}" ]]; then
-    		pushd ./"${REPO_NAME}"
-    		git fetch
-    		git checkout "${HASH}"
-    		popd
-    	fi
-        # Change to the repository directory
-        # Check if the build script exists and is executable
-        if [[ -x "./${REPO_NAME}/scripts/build.sh" ]]; then
-          (
-            echo "Running build script for ${REPO}..."
-            cd "${REPO_NAME}"
-            ./scripts/build.sh
-            echo "Build script for ${REPO} complete!"
-          )
-        else
-            echo "No build script found in ${REPO}"
-        fi
-
-        OBOT_SERVER_VERSIONS="$(cat <<VERSIONS
-"${REPO}": "$(cd "${REPO_NAME}" && git rev-parse --short HEAD)"
+REPO=github.com/obot-platform/tools
```

Comment:
all the same questions from the enterprise-tools repo regarding cutting out cruft in this file